src/composables/map/ol.synchronizer.ts

```diff
@@ -14,6 +14,7 @@ import useMvtStyles from '@/composables/mvt-styles/mvt-styles.composable'
 export class OlSynchronizer {
   previousLayers: Layer[]
   previousVectorSources: VectorSourceDict
+  timeoutID: NodeJS.Timeout
```

Comment:
Maybe change to number as we are not in a server side project.
Also let timeoutID: NodeJS.Timeout need to be change in `src/services/utils/debounce()`